CMakeLists.txt

```diff
@@ -31,4 +31,6 @@ set(IPTV_HEADERS src/client.h
 
 build_addon(pvr.iptvsimple IPTV DEPLIBS)
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DIPTV_VERSION=${IPTV_VERSION}")
```

Comment:
Better use `add_definitions(-DIPTV_VERSION=${IPTV_VERSION})` instead of fiddle with `CMAKE_CXX_FLAGS`.
Also it should be move above `build_addon(...)`

Reply:
Fixed. I need to add call `addon_version()` to actually set the version.